src/highlight.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+export const HighlightButton = () => {
+  return (
+    <button id="highlightBtn"></button>
+  );
+}
+
+export const startHighlight = ()  => {
```

Comment:
Please move the highlight button click handling to here (using an onClick prop) instead of in index.tsx (lines 16-21), and then startHighlight shouldn't need to be exported.